### PR TITLE
added daytona sample courier manager

### PR DESCRIPTION
**Pull request by rupacesigdel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,110 @@
...
-   ```bash  
-   daytona create <SAMPLE_REPO_URL> 
-   ```  
+## Installation
 
-... MORE STEPS IF NEEDED ...
+1. **Create the Daytona Environment**:
+Initialize the project directly with Daytona:
+  ```sh
+    daytona create https://github.com/rupacesigdel/COURIER_WEB.git
```
Comment (Tpuljak) [inline README.md]: URL is wrong. Should be https://github.com/daytonaio/sample-django-courier-manager

**Review by Tpuljak — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rupacesigdel) [reply README.md]: I  updated it . Please review.

**Comment by rupacesigdel** [comment]
done

File: README.md (hunk 1 above)
Comment (rupacesigdel) [reply README.md]: @Tpuljak , Did you review it?

**Pull request merged**